src/stories/EkoVideo.stories.js

```diff
@@ -85,12 +86,52 @@ NoAutoplay.args = {
     }
 };
 
+const cookshopDefaultArgs = {
+    id: "sc88q49",
+    embedAPI: "2.0",
+    excludePropagatedParams: ['id'],
+    ...(
+        isChromatic() &&
+        {
+            // Pause once decision has started (UI is shown)
+            events: {
+                'decision.start': () => {},
+                seeked: function(){
+                    this.once('decision.start', () => {
+                        this.pause();
+                    });
+                },
+            },
+            params: {
+                headnodeid: 'node_0_loop_55ad71',
+                context: 'cc5jqNEz',
+
+                // Disable ekoshell background color when paused
+                deliveryobject: JSON.stringify({
+                    environment: {
+                        playerOptionsOverrides: {
+                            plugins: {
+                                ekoshell: {
+                                    addons: {
+                                        projectmetadata: {
+                                            background: 'rgba(0,0,0,0)'
+                                        }
```

Comment:
@Melatonin64 the subtitles tests don't need this?

Reply:
Not currently, as they run on embed API v1.0 (older delivery).